lightning/src/chain/keysinterface.rs

```diff
@@ -195,8 +195,8 @@ impl Readable for SpendableOutputDescriptor {
 // TODO: We should remove Clone by instead requesting a new ChannelKeys copy when we create
 // ChannelMonitors instead of expecting to clone the one out of the Channel into the monitors.
 pub trait ChannelKeys : Send+Clone {
-	/// Gets the commitment seed
-	fn commitment_seed(&self) -> &[u8; 32];
+	/// Gets the commitment seed for a specific commitment number
```

Comment:
Can you note that the "commitment number" here starts at (1 << 48) - 1 and counts backwards for clarity.

Reply:
done